verilog/formatting/formatter.cc

```diff
@@ -266,12 +267,65 @@ verible::util::Status Formatter::Format(const ExecutionControl& control) {
   const auto unwrapped_lines =
       MakeUnwrappedLinesWorklist(*format_tokens_partitions, style_);
 
+  if (control.show_token_partition_policy) {
+    control.Stream() << "Token partition policy:\n";
+    for (const auto& uwline : unwrapped_lines) {
+      control.Stream() << " :: " << uwline << " :: " << uwline.PartitionPolicy() << "\n";
+    }
+    return verible::util::OkStatus();
+  }
```

Comment:
Suggest we just include partition policy information at each node during the partition tree printing, rather than introduce a separate execution control point.

Reply:
Done